auth-tokens/src/test/java/com/palantir/tokens/auth/BearerTokenTests.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * (c) Copyright 2016 Palantir Technologies Inc. All rights reserved.
```

Comment:
odd that it didn't update the year

Reply:
I have a handy sed command that doesn't update the year since we shouldn't